src/poliastro/tests/kernels/pck00010.tpc

```diff
@@ -0,0 +1,4061 @@
+KPL/PCK
```

Comment:
I wonder if there's a way we can download this kernel on-demand, or at least know the precise URL.

Reply:
The kernel can easily be downloaded on demand using requests. Once the tests are done should the kernel file be deleted?

Reply:
> Once the tests are done should the kernel file be deleted?

Nah, not a big deal.